build/azure-pipelines/sql-product-build.yml

```diff
@@ -72,6 +73,7 @@ stages:
           # Do not run tests for universal build
           RUN_TESTS: false
           RUN_SMOKE_TESTS: false
+        condition: and(succeeded(), eq(variables['VSCODE_BUILD_MACOS_ARM64'], 'true'))
```

Comment:
You shouldn't need this since it already depends on macOS_Universal (which won't run with that's false)

Same with the below. Just keeps it a bit simpler.

Reply:
done